com.brosix.Brosix.json

```
@@ -0,0 +1,72 @@
+{
+    "app-id": "com.brosix.Brosix",
+    "runtime": "org.freedesktop.Platform",
+    "runtime-version": "19.08",
+    "sdk": "org.freedesktop.Sdk",
+    "command": "Brosix.sh",
+    "finish-args": [
+        "--share=ipc",
+        "--share=network",
+        "--socket=pulseaudio",
+        "--socket=x11",
+        "--socket=wayland",
+        "--device=all",
+        "--filesystem=home:rw"
+    ],
+    "modules": [
+	{
+            "name": "v4l",
+	    "config-opts": [ "--disable-static", "--disable-libdvbv5" ],
+	    "no-make-install": true,
+	    "cleanup": [ "/include" ],
+	    "post-install": [
+		"cd lib/libv4lconvert;make install",
+		"cd lib/libv4l2;make install"
+	    ],
+            "sources": [
+                {
+                    "type": "archive",
+                    "url": "https://linuxtv.org/downloads/v4l-utils/v4l-utils-1.12.8.tar.bz2",
+                    "sha256": "cae7a55a6a1330e7b4297190847793d501b6f3e2248fb9571ab895034ca0aebc"
+                }
+            ]
+        },
+        {
+            "name": "Brosix",
+            "buildsystem": "simple",
+            "build-commands": [
+                "mkdir -p /app/share/applications;mv com.brosix.Brosix.desktop /app/share/applications",
+                "mkdir -p /app/share/icons/hicolor/64x64/apps;mv com.brosix.Brosix.png /app/share/icons/hicolor/64x64/apps",
+                "mkdir -p /app/share/appdata;mv com.brosix.Brosix.appdata.xml /app/share/appdata",
```

New appdata files should use metainfo as an extension and should be installed under $datadir/metainfo
```suggestion
                "mkdir -p /app/share/metainfo;mv com.brosix.Brosix.metainfo.xml /app/share/metainfo",
```
Also, just use install here instead of mkdir & mv.